harness/determined/exec/gc_checkpoints.py

```diff
@@ -22,6 +22,8 @@ def delete_checkpoints(
     logging.info("Deleting {} checkpoints".format(len(to_delete)))
 
     for storage_id in to_delete:
+        if storage_id == "":
```

Comment:
why was this necessary? fine as a precaution but in general i would expect this validation to happen earlier

Reply:
true! will add a precaution earlier in cli/checkpoints.py to check if it's a valid uuid.